packages/web-twig/src/Resources/components/FileUploader/FileUploaderInput.twig

```diff
@@ -85,15 +85,11 @@
                 {{- _dragAndDropText -}}
             </span>
         </label>
-        {% if _helperText or _unsafeHelperText %}
-            <div class="{{ _helperTextClassName }}">
-                {% if _helperText %}
-                    {{ _helperText }}
-                {% else %}
-                    {{ _unsafeHelperText | raw }}
-                {% endif %}
-            </div>
-        {% endif %}
+        <HelperText
+            className="{{ _helperTextClassName }}"
```

Comment:
Why HelperText support className instead UNSAFE_className?

Reply:
🤔 Maybe because it is for internal usage and the class is static from the parent component. But this suggests that we need to refactor our styles as well because helper text class should be one and only and not for every form component with component name prefix.

Reply:
What a great question @tomasjanicek! 👍🏻  🙂

As @literat pointed out, `HelperText` and `ValidationText` components are still considered "private". As we don't encourage anyone to use them (yet), we are bold enough to assume the `className` has been injected by another Spirit component and not by a user, and therefore it's always "safe".

But we realize it's a double standard. I suppose it will be all cleaned up once the private components in questions will become public components with a standardized API.

Reply:
I pointed this out in the `Field` directory `README`:

```md
# Form Field Subcomponents

⚠️ All components in this directory are private and should not be used outside of Spirit components.
```